library-compose/build.gradle.kts

```diff
@@ -55,9 +56,16 @@ dependencies {
     implementation(libs.compose.material)
 
     testImplementation(libs.junit)
-    
+
     debugImplementation(libs.compose.ui.test.manifest)
     androidTestImplementation(libs.compose.ui.test.junit4)
     androidTestImplementation(libs.androidx.test.runner)
     androidTestImplementation(libs.androidx.test.ext.junit)
 }
+
+tasks {
+    withType<Detekt>().configureEach {
```

Comment:
This is unnecessary

Reply:
Without this I get the following error:
```
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':library-compose:detekt'.
> Invalid value (20) passed to --jvm-target, must be one of [1.6, 1.8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18]
```

Reply:
That's because you're using JDK 20 which is unsupported with this template at the moment

Reply:
I've documented this in README.md. I had JDK 20 on my PATH which meant that even when I set JDK 17 within IDEA, Gradle was still using JDK 20.